base/distributed/macros.jl

```diff
@@ -16,11 +16,52 @@ spawnat(p, thunk) = sync_add(remotecall(thunk, p))
 
 spawn_somewhere(thunk) = spawnat(nextproc(),thunk)
 
+"""
+    @spawn
+
+Create a closure around an expression and run it on an automatically-chosen process,
+returning a [`Future`](@ref) to the result.
+
+# Examples
+```julia-repl
+julia> addprocs(3);
+
+julia> f = @spawn myid()
+Future(2, 1, 5, Nullable{Any}())
+
+julia> fetch(f)
+2
+
+julia> f = @spawn myid()
+Future(3, 1, 7, Nullable{Any}())
+
+julia> fetch(f)
+3
+```
+"""
 macro spawn(expr)
     thunk = esc(:(()->($expr)))
     :(spawn_somewhere($thunk))
 end
 
+"""
+    @spawnat
+
+Create a closure around an expression and run the closure
+asynchronously on process `p`. Returns a [`Future`](@ref) to the result.
+Accepts two arguments, `p` and an expression.
```

Comment:
Perhaps augmenting the signature (rather than describing the signature in the docstring body) would be more consistent with other docstrings?